### PR TITLE
Added new quote by Swami Vivekananda

### PR DESCRIPTION
**Pull request by reachtokish**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -271,6 +271,10 @@
     "author": "Jules Michelet",
     "sentence": "The most intimate temper of a people, its deepest soul, is above all in its language."
   },
+  {
+    "author": "Swami Vivekananda",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:275]: Please add it alphabetically under the quotes starting with the author name of "S".

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Hey @reachtokish thanks for the PR, update the changes as requested.

**Comment by reachtokish** [comment]
> Hey @reachtokish thanks for the PR, update the changes as requested.

Done. Sorry I missed that part.

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @reachtokish!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**